Fix/586 remove WP Rocket banner when plugin installed (but inactive)

markonikolic985 opened the pull request:
Closes #586 

Hide WP Rocket Banner when plugin is installed but not active.

Changed files:

- [views/page-settings.php](https://github.com/wp-media/imagify-plugin/blob/88e39cf23b75b2e63f253e6e6fcd996e4986935b/views/page-settings.php#L16) to keep `imagify-have-rocket` CSS class so the main column has 100% width
- [views/page-rocket-ad.php](https://github.com/wp-media/imagify-plugin/blob/88e39cf23b75b2e63f253e6e6fcd996e4986935b/views/part-rocket-ad.php#L4-L8) to check if plugin exists (active or not)

markonikolic985 committed:
hide rocket banner

Hide WP Rocket Banner when plugin is installed but not active

markonikolic985 committed:
coding standard fix

Mai-Saad commented:
Can see the fix on the settings page (currently if WPR is installed or activated, the Banner is not displayed). However, Can't check the Bulk Optimization page as after closing the banner while exploring on the trunk, it didn't appear again even after deleting transients.

**Update:** Tried another account, Banner removed from settings page but not from Bulk Optimization page. @markonikolic985 Can you please check this case?
https://jmp.sh/Un0UdNm

iCaspar committed:
Remove extra invisible characters

iCaspar committed:
Revise condition for wp-rocket banner on bulk page

We'll check that the plugin is installed, even if it is not currently active to determine whether to display the banner.

iCaspar commented:
Hey here @Mai-Saad 
I've added a tweak to hid the banner on the bulk page as well.
Also, if you previously dismissed a banner, you will find in the db -> wp_usermeta table an entry for `_imagify_ignore_ads` that stores the dismissed status. If you remove that, it should reset the banner to the "as new" state.

GeekPress reviewed (approved):
QA done, it's working like expected :)
